src/components/backend-ai-session-list.ts

```diff
@@ -1118,7 +1118,7 @@ export default class BackendAiSessionList extends BackendAIPage {
             <wl-button fab flat inverted class="fg controls-running"
                                @click="${(e) => this._runTerminal(e)}"><wl-icon>keyboard_arrow_right</wl-icon></wl-button>
           ` : html``}
-          ${(this._isRunning && !this._isPreparing(rowData.item.status)) || this._APIMajorVersion > 4 ? html`
+          ${(this._isRunning && !this._isPreparing(rowData.item.status)) ? html`
```

Comment:
Good. We now do not need to support the legacy API (version < 5) in the main branch. But, the destroy button should be displayed for `ERROR` sessions to force-terminate them. So, please update the line like following:

```javascript
${(this._isRunning && !this._isPreparing(rowData.item.status)) || this._isError(rowData.item.status) ? html`
```

`_isError` function might look like:

```javascript
_isError(status) {
  return status === 'ERROR';
}
```

Good location for this function would be after `_isPreparing` function.